modules/engines/engine-hdb/src/test/java/com/sap/xsk/hdb/ds/test/parser/XSKHdbddParserTest.java

```diff
@@ -573,4 +573,17 @@ public void testParseHDBDDWithViewDefinitionWithUnion() throws Exception {
 
     assertEquals(expectedRawContent, viewModel.getRawContent().trim());
   }
+
+  @Test
+  public void testParseHDBDDWithUniqueCatalogIndex() throws Exception {
+    XSKDataStructureModel parsedModel = XSKDataStructureModelFactory.parseHdbdd("gstr2/CatalogIndexUnique.hdbdd", "");
+    assertFalse(((XSKDataStructureCdsModel) parsedModel).getTableModels().get(0).getConstraints().getUniqueIndices().isEmpty());
```

Comment:
Same comment for the argument of `assertFalse` as for the `assertNull` previously